main/createUiDefinition.json

```diff
@@ -69,6 +69,18 @@
           "validationMessage": "Insert valid Jointoken."
         },
         "visible": "true"
+      },
+      {
+        "name": "accessproxy",
+        "type": "Microsoft.Common.TextBox",
+        "label": "http proxy",
+        "toolTip": "Pass Accessproxy.",
```

Comment:
Can we change these tool tip description . we can use text from bootstrap ui for reference